Well I've compiled gmsh, I've installed the latest version from PyPi, gmsh works fine as a binary.

But I, for the life of me, cannot get the FEM module to load:

```
14:25:32  DLL load failed while importing Fem: The specified procedure could not be found.
14:26:06  Cannot create object 'Box_Mesh': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'FemConstraintFixed': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'FemConstraintForce': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'Analysis': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'CalculiXccxTools': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'WarpVector': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'Pipeline': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'ResultMesh': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  Cannot create object 'CCX_Results': (DLL load failed while importing Fem: The specified procedure could not be found.)
14:26:06  <PropShape> PropertyTopoShape.cpp(416): Pending recompute for generating element map: FemCalculixCantilever3D#Box
14:26:06  Traceback (most recent call last):
  File "C:\Program Files\FreeCAD\bin\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "C:\Program Files\FreeCAD\Mod\Fem\femguiutils\migrate_gui.py", line 163, in exec_module
    return self.load_module(module)
  File "C:\Program Files\FreeCAD\Mod\Fem\femguiutils\migrate_gui.py", line 210, in load_module
    import femviewprovider.view_material_common
  File "C:\Program Files\FreeCAD\bin\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "C:\Program Files\FreeCAD\Mod\Fem\femviewprovider\view_material_common.py", line 37, in <module>
    from . import view_base_femconstraint
  File "C:\Program Files\FreeCAD\bin\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "C:\Program Files\FreeCAD\Mod\Fem\femviewprovider\view_base_femconstraint.py", line 35, in <module>
    from femviewprovider import view_base_femobject
  File "C:\Program Files\FreeCAD\bin\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
  File "C:\Program Files\FreeCAD\Mod\Fem\femviewprovider\view_base_femobject.py", line 38, in <module>
    import FemGui  # needed to display the icons in TreeView
  File "C:\Program Files\FreeCAD\bin\Lib\site-packages\shiboken2\files.dir\shibokensupport\__feature__.py", line 142, in _import
    return original_import(name, *args, **kwargs)
<class 'ImportError'>: DLL load failed while importing FemGui: The specified procedure could not be found.
```

I've attached all manner of debugger I can think of, but it's failing to load some kind of symbol/method from a DLL. It doesn't tell me **which** it's failing to load, which is frustrating.. But I've copied every DLL I can possibly find that was generated by FreeCAD or its dependencies, and nothing is working. 

In x64dbg, the `ENTRY_POINT_NOT_FOUND` status being emitted always seems to be associated with an address the symbol name `?Perform@BOPAlgo_Builder@@UEAAXXZ` stored in register R12... but the symbol exists in the OCCT libraries compiled. So I don't know what it's complaining about.

Perhaps OCCT itself is trying to load another DLL, but it's not telling me which.

Overall quite frustrating...

--- a/src/3rdParty/salomesmesh/src/NETGENPlugin/NETGENPlugin_Mesher.cpp
+++ b/src/3rdParty/salomesmesh/src/NETGENPlugin/NETGENPlugin_Mesher.cpp
@@ -66,6 +66,9 @@
 #include <TopTools_DataMapOfShapeShape.hxx>
 #include <TopTools_MapOfShape.hxx>
 #include <TopoDS.hxx>
+#include <BRepBndLib.hxx>
+#include <BRepMesh_IncrementalMesh.hxx>
+#include <TopoDS_Solid.hxx>
 
 #ifdef _MSC_VER
 #pragma warning(disable : 4067)

--- a/src/3rdParty/salomesmesh/src/NETGENPlugin/NETGENPlugin_NETGEN_2D_ONLY.cpp
+++ b/src/3rdParty/salomesmesh/src/NETGENPlugin/NETGENPlugin_NETGEN_2D_ONLY.cpp
@@ -44,6 +44,8 @@
 #include <Standard_ErrorHandler.hxx>
 #include <Standard_Failure.hxx>
 
+#include <TopExp.hxx>
+
 #include <utilities.h>
 
 #include <list>

--- a/src/App/StringHasher.cpp
+++ b/src/App/StringHasher.cpp
@@ -26,6 +26,7 @@
 #ifndef _PreComp_
 #endif
 
+#include <deque>
 #include <boost/io/ios_state.hpp>
 #include <boost/iostreams/device/array.hpp>
 #include <boost/iostreams/stream.hpp>

